examples/gitlab/gitlab-aws.yaml

```diff
@@ -0,0 +1,60 @@
+name: gitlab
+org_name: <<org_name::org_name::Your Organization Name>>
+providers:
+  aws:
+    region: <<region::aws_region::Your AWS Region>>
+    account_id: <<account_id::aws_account_id::Your AWS Account ID>>
+modules:
+  - type: base
+  - type: dns
+    domain: <<domain::string::The dns domain for your airflow deployment>>
+    delegated: false
+  - type: k8s-cluster
+  - type: k8s-base
+  # - type: postgres
+  #   name: gitlabpg
+  # - type: redis
+  #   name: gitlabredis
+  - name: gitlab
+    type: helm-chart
+    chart: gitlab
+    create_namespace: true
+    namespace: "gitlab"
+    repository: "https://charts.gitlab.io"
+    chart_version: "5.8.2"
+    values:
+      postgresql:
+        master:
+          podAnnotations:
+            linkerd.io/inject: enabled
+            config.linkerd.io/proxy-cpu-limit: "0.1"
+            config.linkerd.io/proxy-cpu-request: "0.05"
+            config.linkerd.io/proxy-memory-limit: "20Mi"
+            config.linkerd.io/proxy-memory-request: "10Mi"
```

Comment:
```suggestion
```

These values are now set by default, so the user doesn't need to set these explicitly.